migrate/v0_11/migrate.go

```diff
@@ -65,3 +67,46 @@ func MigrateBep3(oldGenState v0_9bep3.GenesisState) v0_11bep3.GenesisState {
 		PreviousBlockTime: v0_11bep3.DefaultPreviousBlockTime,
 	}
 }
+
+// MigrateIncentive migrates from a v0.9 (or v0.10) incentive genesis state to a v0.11 incentive genesis state
+func MigrateIncentive(oldGenState v0_9incentive.GenesisState) v0_11incentive.GenesisState {
+	var newRewards v0_11incentive.Rewards
+	var newRewardPeriods v0_11incentive.RewardPeriods
+	var newClaimPeriods v0_11incentive.ClaimPeriods
+	var newClaims v0_11incentive.Claims
+	var newClaimPeriodIds v0_11incentive.GenesisClaimPeriodIDs
+
+	newMultiplier := v0_11incentive.NewMultiplier(v0_11incentive.Large, 12, sdk.OneDec())
+
+	for _, oldReward := range oldGenState.Params.Rewards {
+		newReward := v0_11incentive.NewReward(oldReward.Active, oldReward.Denom+"-a", oldReward.AvailableRewards, oldReward.Duration, v0_11incentive.Multipliers{newMultiplier}, oldReward.ClaimDuration)
+		newRewards = append(newRewards, newReward)
+	}
+	newParams := v0_11incentive.NewParams(true, newRewards)
+
+	for _, oldRewardPeriod := range oldGenState.RewardPeriods {
+
+		newRewardPeriod := v0_11incentive.NewRewardPeriod(oldRewardPeriod.Denom+"-a", oldRewardPeriod.Start, oldRewardPeriod.End, oldRewardPeriod.Reward, oldRewardPeriod.ClaimEnd, v0_11incentive.Multipliers{newMultiplier})
+		newRewardPeriods = append(newRewardPeriods, newRewardPeriod)
+	}
+
+	for _, oldClaimPeriod := range oldGenState.ClaimPeriods {
+		newClaimPeriod := v0_11incentive.NewClaimPeriod(oldClaimPeriod.Denom+"-a", oldClaimPeriod.ID, oldClaimPeriod.End, v0_11incentive.Multipliers{newMultiplier})
+		newClaimPeriods = append(newClaimPeriods, newClaimPeriod)
+	}
+
+	for _, oldClaim := range oldGenState.Claims {
+		newClaim := v0_11incentive.NewClaim(oldClaim.Owner, oldClaim.Reward, oldClaim.Denom+"-a", oldClaim.ClaimPeriodID)
+		newClaims = append(newClaims, newClaim)
+	}
+
+	for _, oldClaimPeriodID := range oldGenState.NextClaimPeriodIDs {
+		newClaimPeriodID := v0_11incentive.GenesisClaimPeriodID{
+			CollateralType: oldClaimPeriodID.Denom + "-a",
+			ID:             oldClaimPeriodID.ID,
+		}
+		newClaimPeriodIds = append(newClaimPeriodIds, newClaimPeriodID)
+	}
```

Comment:
There's a ukava claim_period_id in the testdata that gets converted to ukava-a. Do you know if that will cause problems? I can't think of any.

Reply:
Goofy, but won't cause an issue - don't see a need to delete it.